src/api/taxonomyApi.ts

```diff
@@ -185,10 +185,7 @@ export async function queryTopicsOnContentURI(
   id: string,
   context: Context,
 ): Promise<GQLTopic> {
-  const response = await fetch(
-    `/taxonomy/v1/queries/topics?contentURI=${id}`,
-    context,
-  );
+  const response = await fetch(`/taxonomy/v1/topics?contentURI=${id}`, context);
```

Comment:
skal includeMetadata være med her og?

Reply:
Tja. Disse kalla har vore for å filtrere ut ting som ikkje skal vises, så vi bør sikkert skrive dei litt om no som vi har metadata tilgjengelig. Skal sjå på det.

Reply:
Nei, forresten. Sjekka og endepunktet brukes for å linke inn filmer på ndla-filmforsida. Her hentes kun ut path for filmene så metadata trengs ikkje.

Reply:
ok :)